server/shared/logger.js

```diff
@@ -1,16 +1,57 @@
 'use strict';
 
-const bunyan = require('bunyan');
+const Bunyan = require('bunyan');
+const pkg = require('../../package.json');
+const safeRequire = require('safe-require');
 
-const logLevel = (() => {
-  if (process.env.LOG_LEVEL) return process.env.LOG_LEVEL;
-  return process.env.NODE_ENV === 'production' ? 'info' : 'debug';
-})();
+const isProduction = process.env.NODE_ENV === 'production';
+const supportsEmoji = process.platform !== 'win32' &&
+  process.env.TERM === 'xterm-256color';
 
-const logger = bunyan.createLogger({
-  name: 'tailor-server',
-  serializers: bunyan.stdSerializers,
-  level: logLevel
-});
+const Level = getLevels(Bunyan);
+const loggers = {};
 
-module.exports = logger;
+class Logger extends Bunyan {
+  addStream(stream, defaultLevel) {
+    if (!isProduction) {
+      stream = getOutputStream(stream);
+    }
+    return super.addStream(stream, defaultLevel);
+  }
+
+  _emit(record, noemit) {
+    return super._emit(record, !createLogger.enabled || noemit);
+  }
+}
+
+function createLogger(name, options = {}) {
+  name = [pkg.name, name].filter(Boolean).join(':');
+  const serializers = { ...Bunyan.stdSerializers, ...options.serializers };
+  if (!loggers[name]) {
+    options.level = process.env.LOG_LEVEL || options.level;
```

Comment:
Unless specified, the default logger level matches `LOG_LEVEL` env var to make sure it gets printed to the stdout.